.github/workflows/branch-sync.yml

```diff
@@ -51,23 +51,22 @@ jobs:
         with:
           fetch-depth: 0
 
+      # Determine which branch to merge into, e.g:
+      # * if a change is made to 1.2.x, merge it into 1.3.x;
+      # * if a change is made to 1.3.x, merge it into 1.4.x;
+      # * if a change is made to 1.4.x, merge it into master.
       - name: Determine base branch
         id: get-base-branch
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          # Determine which branch to merge into, e.g:
-          # * if a change is made to 1.2.x, merge it into 1.3.x;
-          # * if a change is made to 1.3.x, merge it into 1.4.x;
-          # * if a change is made to 1.4.x, merge it into master.
-
           # default to the HEAD branch (typically either master or main)
-          base_branch=$(gh repo view --json defaultBranchRef -q .defaultBranchRef.name)
+          base_branch="$(gh repo view --json defaultBranchRef -q .defaultBranchRef.name)"
```

Comment:
Strange, this worked with the pure `git` command, if `gh repo view` returning results with spaces?

Reply:
(That was just a tidy commit, ingrained shellcheck nitpicking)